docs/repo-docs/crafting-your-repository/configuring-tasks.mdx

```diff
@@ -326,30 +326,30 @@ Some tasks should always be ran no matter what, like a deployment script after a
 
 Some tasks can be ran in parallel despite being dependent on other packages. An example of tasks that fit this description are linters, since a linter doesn't need to wait for outputs in dependencies to run successfully.
```

Comment:
Alternatively you could change the above example to be typechecking instead, but based on this dissonance I'm actually a little confused by your example. Using Internal Packages™

> since a type checker doesn't need to wait for outputs in dependencies to run successfully

could be true. Not using Internal Packages, where the output is `.d.ts` files, it would seem like you definetly can't typecheck package A if package B could be changing `.d.ts` files.